nodepool/config/templates/cluster.yaml

```diff
@@ -46,9 +46,15 @@ availabilityZone: {{.AvailabilityZone}}
 # ARN of the KMS key used to encrypt TLS assets.
 kmsKeyArn: "{{.KMSKeyARN}}"
 
-# Number of worker nodes to create
+# Number of worker nodes to create, for more control use `workerASG` and do not use this setting
 #workerCount: 1
 
+# Auto Scaling Group definition for workers. If only `workerCount` is specified, min and max will be the set to that value and `rollingUpdateMinInstancesInService` will be one less.
+# workerASG:
+#   minSize: 1
+#   maxSize: 3
+#   rollingUpdateMinInstancesInService: 2
```

Comment:
I'm not entire sure how I can get this in without moving `worker.go` since otherwise there will be a duplicate `Worker` inside `Cluster`. Are you ok for me to move it? I saw there was a bunch of placeholder code anyway in `config.go` so it's pretty messy as it is.